gae/article.tpl

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="{{ lang }}">
-  <head>
+  <head dir="auto">
```

Comment:
W3C suggested adding dir to the head (I think they mean ltr/rtl because it was before introducing dir=auto, or it's for older versions of HTML) but it doesn't seem to do any thing on latest versions of chrome & firefox!